src/nuclio/common/components/edit-item/edit-item.component.js

```diff
@@ -290,6 +331,20 @@
             }
         }
 
+        /**
+         * Handler on specific action type of trigger's subscription
+         * @param {string} actionType
+         * @param {number} index - index of variable in array
+         */
+        function handleSubscriptionAction(actionType, index) {
+            if (actionType === 'delete') {
+                ctrl.subscriptions.splice(index, 1);
+                lodash.unset(ctrl.item, 'attributes.subscriptions.' + index);
```

Comment:
Are these both arrays?
Why do you use `splice` for one and `unset` for another?

Another option is to use `_.pullAt` method:
```js
var arr = ['a', 'b', 'c', 'd', 'e', 'f', 'g'];
// index:  0    1    2    3    4    5    6

// one arg
_.pullAt(arr, 2); // removes an item at index 2 of arr
console.log(arr); // prints ['a', 'b', 'd', 'e', 'f', 'g']

// multiple args
_.pullAt(arr, [0, 3, 5]); // removes items at indices 0, 3 and 5 of arr
console.log(arr);         // prints: ['b', 'd', 'f']
```
I'm not telling you to use it here, I'm just letting you know it exists too, and you can choose which method you like to use.